Mathlib/LinearAlgebra/Multilinear/Finsupp.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2025 Daniel Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Morrison
+-/
+module
+
+public import Mathlib.LinearAlgebra.Multilinear.DFinsupp
+
+/-!
+# Interactions between finitely-supported functions and multilinear maps
+
+## Main definitions
+
+* `freeFinsuppEquiv` is an equivalence of multilinear maps over free modules with finitely
+  supported maps.
+
+-/
+
+@[expose] public section
+
+universe uι uκ uS uR uM uN
+variable {ι : Type uι} {κ : ι → Type uκ}
+variable {S : Type uS} {R : Type uR}
```

Comment:
No need to handle universes explicitly and we might as well declare `ι'` while we're here:
```suggestion
variable {ι ι' S R : Type*} {κ : ι → Type*}
```